packages/web-components/src/components/ic-hero/ic-hero.css

```diff
@@ -247,6 +247,11 @@ ic-typography.heading-bottom-spacing {
     min-height: 192px;
   }
 
+  :host,
+  .right-container {
+    margin-bottom: 1rem;
```

Comment:
Can you swap the value for our design token: https://github.com/mi6/ic-ui-kit/blob/main/packages/web-components/src/global/variables.css#L276